src/locale/properties/register/register.en.properties

```diff
@@ -38,19 +38,18 @@ register.additionalEmails=Additional email
 register.additionalEmailsCanBeDuplicated=Additional email cannot match primary email
 register.additionalEmailAdd=Add another email
 register.additionalEmailInfo=Ensure that you never lose access to your ORCID record by registering additional email addresses on your account.
-register.termsRequired=You must accept the terms and conditions
+register.termsRequired=You must accept the terms of use
```

Comment:
Fix property in the Html is called `register.terms`

Reply:
This is is a different one an is only use one `form-terms.component.html`

Reply:
register.terms exist for all testing languages